### ISSUE TITLE
update the "kubeflow training operator" and "mpi operator" docs

### ISSUE THREAD

**Issue by thesuperzapper** [kubeflow/website#3563]
Currently, the [`"kubeflow training operator"` docs](https://www.kubeflow.org/docs/components/training/) are extremely [out of date](https://github.com/kubeflow/website/tree/master/content/en/docs/components/training), and don't even reflect that all the operators are now aggregated into one.

### I propose we do the following:

1. Split the [`"components / training operator"`](https://github.com/kubeflow/website/tree/master/content/en/docs/components/training) section into two:
     - Unified Training Operator: https://github.com/kubeflow/training-operator
     - MPI Operator: https://github.com/kubeflow/mpi-operator
2. Create an "introduction/about" section for each of these new sections, which links to their respective repos and briefly introduces what they do
3. Update the docs to actually reflect how to use the latest versions of these tools

**Comment by thesuperzapper** [comment]
@kubeflow/wg-training-leads I think this needs to be a top priority, because your component is effectively undocumented right now.

**Comment by thesuperzapper** [comment]
@andreyvelich After reading more of the existing docs, I agree with your statements in the community meeting that they are not strictly speaking "out of date". However, I still think they need work, and my proposal in the above issue will help address it.

**Comment by terrytangyuan** [comment]
@tenzen-y @alculquicondor Would any of you like to help update the MPI related documentation?

**Comment by andreyvelich** [comment]
I agree with you @thesuperzapper. As I said on the latest community call, I will try to find some free time to add an overview page for Training Operator in the next few weeks.
/assign @andreyvelich

**Comment by thesuperzapper** [comment]
I belive this has been mostly resolved by https://github.com/kubeflow/website/pull/3651

/close
